src/cma.ts

```diff
@@ -184,17 +184,21 @@ export async function runCumulusTask(
     input: cmaProcess.stdout
   });
 
-  let lambdaTimer = setTimeout(() => {
-    console.log(
-      'Info: No CMA buffer dump set, lambda is not running as a lambda'
-    );
-  }, 0);
+  let lambdaTimer;
+
   if (typeof context.getRemainingTimeInMillis === 'function') {
     lambdaTimer = setTimeout(() => {
       console.log('Lambda timing out, writing CMA stderr to cloudwatch logs');
       console.log(`CMA (on start task timeout) StdErr: \n ${errorObj.stderrBuffer}`);
     }, context.getRemainingTimeInMillis() - 5 * 100);
+  } else {
+    console.log(
+      'Info: No CMA timeout buffer dump set, as lambda is not running as a lambda'
+    );
+    lambdaTimer = setTimeout(() => {
```

Comment:
Will this fix the issue with the stderr not being dumped in the LP DAAC logs?

Reply:
I believe the reason that occurred was because they weren't using the updated lambda layer.       I manually have deployed sync granules and validated I'm getting output as well as a timed-out hello world.